runtimes/politest/src/lib.rs

```diff
@@ -1002,56 +1111,80 @@ construct_runtime!(
+		Random: pallet_insecure_randomness_collective_flip = 52,
 
-		// Polimec Core
-		PolimecFunding: pallet_funding::{Pallet, Call, Storage, Event<T>, Config<T>, HoldReason}  = 52,
-		LinearRelease: pallet_linear_release::{Pallet, Call, Storage, Event<T>} = 53,
-		Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 54,
-
-		// Utilities
-		Random: pallet_insecure_randomness_collective_flip = 60,
-
-
+		// Oracle
+		Oracle: orml_oracle::{Pallet, Call, Storage, Event<T>} = 70,
+		OracleProvidersMembership: pallet_membership::<Instance1> = 71,
+		OracleOffchainWorker: pallet_oracle_ocw::{Pallet, Event<T>} = 72,
 
-		// Among others: Send and receive DMP and XCMP messages.
-		ParachainSystem: cumulus_pallet_parachain_system = 80,
-		ParachainInfo: parachain_info::{Pallet, Storage, Config<T>} = 81,
-		// Wrap and unwrap XCMP messages to send and receive them. Queue them for later processing.
-		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 82,
-		// Build XCM scripts.
-		PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin, Config<T>} = 83,
-		// Does nothing cool, just provides an origin.
-		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 84,
-		// Queue and pass DMP messages on to be executed.
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 85,
-		MessageQueue: pallet_message_queue = 86,
+		// Funding - this should go after Oracle
+		Funding: pallet_funding = 80,
+		LinearRelease: pallet_linear_release = 81,
 	}
 );
 
 #[cfg(feature = "runtime-benchmarks")]
 mod benches {
-	define_benchmarks!(
+	frame_benchmarking::define_benchmarks!(
+		// System support stuff.
 		[frame_system, SystemBench::<Runtime>]
+		[pallet_timestamp, Timestamp]
+		[pallet_utility, Utility]
+		[pallet_multisig, Multisig]
+		[pallet_proxy, Proxy]
+
+		// Monetary stuff.
 		[pallet_balances, Balances]
+		[pallet_vesting, Vesting]
+
+		// Collator support.
 		[pallet_session, SessionBench::<Runtime>]
-		[pallet_timestamp, Timestamp]
+		[pallet_parachain_staking, ParachainStaking]
+
+		// XCM helpers.
 		[cumulus_pallet_xcmp_queue, XcmpQueue]
-		[pallet_funding, PolimecFunding]
+		[pallet_xcm, PalletXcmExtrinsiscsBenchmark::<Runtime>]
+
+		// Governance
+		[pallet_treasury, Treasury]
+		[pallet_democracy, Democracy]
+		[pallet_collective, Council]
+		[pallet_collective, TechnicalCommittee]
+		[pallet_elections_phragmen, Elections]
+		[pallet_preimage, Preimage]
+		[pallet_scheduler, Scheduler]
+
+		// Oracle
+		[pallet_membership, OracleProvidersMembership]
+		//[orml_oracle, Oracle]
+		// Funding
+		[pallet_funding, Funding]
 		[pallet_linear_release, LinearRelease]
 	);
 }
 
 impl_runtime_apis! {
 	impl sp_consensus_aura::AuraApi<Block, AuraId> for Runtime {
 		fn slot_duration() -> sp_consensus_aura::SlotDuration {
-			sp_consensus_aura::SlotDuration::from_millis(Aura::slot_duration())
+			sp_consensus_aura::SlotDuration::from_millis(parachains_common::SLOT_DURATION)
```

Comment:
What does this change?

Reply:
Just to be fully aligned with https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-1---update-parachain-runtime